### ISSUE TITLE
"multiple versions of module ..." message doesn't disappear once fixed

### ISSUE THREAD

**Issue by robertgarrigos** [backdrop/backdrop-issues#5702]
I usually have all my contrib modules under `contrib` folder, under `modules` folder. But I installed, by error, one module under `modules`, so I ended up with two versions of the same module as it was already under `contrib`. Once alerted by the status report page, I deleted the duplicated modules, but the message still appears in the status page. 

I checked in the `system` table how many entries were for that module. It was fine, only the right one.

I checked in the file system. There was no extra module where the report still says there is one. I also checked the permissions of the file system.

I run cron, update.php and cleared the cache, and still the status report page shows I have a duplicated module, and it's pointing me to a directory where there is no extra module.

**Comment by indigoxela** [comment]
> I deleted the duplicated modules, but the message still appears in the status page.

Were there several? Which ones?

> ... and still the status report page shows I have a duplicated module, and it's pointing me to a directory where there is no extra module.

That seems odd. Can you check in your system database table for entries for your modules? (Impudently assuming that you have access to the database :wink: )

Because the information about installed modules gets pulled from there. So, if you simply can't get rid of the message, no matter how you kick the cache tires, there might be an inconsistency in that table.

Most of all: your bug report is a bit sparse, can you please post the _complete status message_ about duplicate modules?

**Comment by argiepiano** [comment]
It may be helpful to know which module is causing this notice.

**Comment by robertgarrigos** [comment]
It is feeds modules.

![Informe_d_estat___codi_garrigos_tech](https://user-images.githubusercontent.com/105581/182319877-439e266e-677e-425d-8292-4ded3fa8fbc0.jpg)

As I said, there is not entry on system table about any feed module under `modules`. Only the ones under `contrib`:
![codi_garrigos_tech___codi_-_MySQL_8_0_30](https://user-images.githubusercontent.com/105581/182320302-e761b69a-0a58-4852-bc0c-694275119255.jpg)

**Comment by indigoxela** [comment]
I tried the following to reproduce:

1. Installed feeds and job_scheduler in a "contrib" subfolder
2. Enabled both and checked for problems on the status - no problems
3. Copied the feeds folder to the modules folder
4. Verified that there are warnings on the status page
5. Removed the second feeds folder
6. Verified that the warnings are gone on the status page (immediately)

@robertgarrigos whatever is fishy or went wrong with your install - I can not reproduce it. Any idea which step might be important?
